code/modules/clothing/ears/earmuffs.dm

```diff
@@ -27,12 +27,24 @@
 		return
 
 	if(headphones_on)
-		icon_state = "headphones_off"
+		icon_state = initial(icon_state)
 		headphones_on = 0
 		to_chat(usr, "<span class='notice'>You turn the music off.</span>")
 	else
-		icon_state = "headphones_on"
+		icon_state = "[initial(icon_state)]_on"
 		headphones_on = 1
 		to_chat(usr, "<span class='notice'>You turn the music on.</span>")
 
-	update_clothing_icon()
\ No newline at end of file
+	update_clothing_icon()
+
+/obj/item/clothing/ears/earmuffs/headphones/earphones
+	name = "earphones"
+	var/earphones_on = 0
```

Comment:
This seems to already be taken care of by `headphones_on`

Reply:
An oversight from when I was trying to figure out how to make it, var/earphones_on has been deleted.